sdk/keyvault/keyvault/CMakeLists.txt

```diff
@@ -29,4 +29,8 @@ add_executable (az_keyvault_test test/main.c)
 
 target_link_libraries(az_keyvault_test PRIVATE az_keyvault)
 
```

Comment:
In this file, you'll need to add " -Wno-missing-field-initializers -Wno-missing-braces" so that `={ 0 }` syntax works on Mac (see azure core's cmakelists.txt)